docs/get-started/usage-ui.md

```diff
@@ -0,0 +1,141 @@
+
+...
+
+```
+
+From the code, you can set the composable code to that view as follows:
+```kotlin
+
+class ExampleActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContentView(R.layout.activity_main)
+        findViewById<ComposeView>(R.id.gravatarComposeView).setContent {
+            GravatarProfileSummary()
+        }
+    }
+}
+
+
+@Composable
+fun GravatarProfileSummary(emailAddress: String = "gravatar@automattic.com") {
+    // Create a ProfileService instance
+    val profileService = ProfileService()
+
+    // Set the default profile state to loading
+    var profileState: ComponentState<Profile> by remember { mutableStateOf(ComponentState.Loading, neverEqualPolicy()) }
+
+    // We wrap the fetch call in a LaunchedEffect to fetch the profile when the composable is first launched, but this
+    // could be triggered by a button click, a text field change, etc.
+    LaunchedEffect(emailAddress) {
+        // Set the profile state to loading
+        profileState = ComponentState.Loading
+        // Fetch the user profile
+        when (val result = profileService.retrieve(Email(emailAddress))) {
+            is GravatarResult.Success -> {
+                // Update the profile state with the loaded profile
+                result.value.let {
+                    profileState = ComponentState.Loaded(it)
+                }
+            }
+            is GravatarResult.Failure -> {
+                // An error can occur when a profile doesn't exist, if the phone is in airplane mode, etc.
+                // Here we log the error, but ideally we should show an error to the user.
+                Log.e("Gravatar", result.error.name)
+                // Set the Empty state on error
+                profileState = ComponentState.Empty
+            }
+        }
+    }
+
+    // Show the profile as a ProfileCard
+    ProfileSummary(profileState, modifier = Modifier.fillMaxWidth().padding(16.dp))
+}
+```
+
+More information on the official documentation: [Using Compose in Views](https://developer.android.com/develop/ui/compose/migrate/interoperability-apis/compose-in-views).
+
+### Override the GravatarTheme
+
+You're free to customize the Gravatar profile component to fit your app's design. You can do this by overriding the `GravatarTheme` in your app's theme.
+
+```kotlin
+CompositionLocalProvider(LocalGravatarTheme provides object : GravatarTheme {
+    // Override theme colors
+    override val colorScheme: ColorScheme
+        @Composable
+        get() = MaterialTheme.colorScheme.copy(outline = Color.LightGray)
+
+    // Override typography style
+    override val typography: Typography
+        @Composable
+        get() = MaterialTheme.typography.copy(
+            headlineSmall = MaterialTheme.typography.headlineSmall.copy(fontWeight = FontWeight.ExtraBold),
+        )
+}) {
+    LargeProfileSummary(profile = userProfile)
+}
+```
```

Comment:
❓ We've added a section with the Android Permissions in the QE section. Should we do the same here and in the core file? I know here it's just the INTERNET permission, but to keep the same structure.

Reply:
INTERNET is not a runtime permission so I thought it was not that important, but it won't hurt!

Reply:
Yeah, please feel free to ignore the comment. Not important at all. 🙂